server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -896,6 +903,14 @@ public void relocated(
             // Fail primary relocation source and target shards.
             failShard("timed out waiting for relocation hand-off to complete", null);
             throw new IndexShardClosedException(shardId(), "timed out waiting for relocation hand-off to complete");
+        } catch (Exception ex) {
+            logger.warn("exception occurred during relocation hand-off to complete errorMsg={}", ex.getMessage());
```

Comment:
logger.error("Exception occurred during relocation hand-off to complete", ex);

Reply:
Removing this log line as we are throwing the same exception as is. I have verified that the original exception shows up from the logger of `IndicesClusterStateService`. Having same log with complete stack trace twice will only litter the code and logs.

Reply:
resolving this.